src/components/EditorTopBar.tsx

```diff
@@ -16,15 +17,27 @@ export const EditorTopBar = ({
     appConfiguration: { dopplerExternalUrls },
   } = useAppServices();
 
+  const intl = useIntl();
+  const translate = (id?: any, values?: any) =>
+    intl.formatMessage({ id: id }, values);
+
   return (
     <div className="editor-top-bar" {...otherProps}>
       <ul className="ed-header-list">
         <li>
-          <DropdownButton buttonText="Salir del Editor">
-            <a href={dopplerExternalUrls.home}>Inicio</a>
-            <a href={dopplerExternalUrls.campaigns}>Campañas</a>
-            <a href={dopplerExternalUrls.lists}>Listas</a>
-            <a href={dopplerExternalUrls.controlPanel}>Panel de Control</a>
+          <DropdownButton buttonText={translate("topbar_exit_button")}>
+            <a href={dopplerExternalUrls.home}>
+              {translate("topbar_exit_button_home")}
```

Comment:
In my opinion, it is not too verbose:

```suggestion
              {intl.formatMessage({ id: "topbar_exit_button_home" })}
```

Reply:
👍

Reply:
Applied